syntax = "proto3";

import "Common.proto";

message EnterRoomRequest
{
    int64   id = 1;
    string  password = 2;
}

message ChatNotify
{
    int64   id = 1;
    string  message = 2;
}
